app/packages/looker/src/lookers/abstract.ts

```diff
@@ -730,7 +776,20 @@ export abstract class AbstractLooker<
       sources: this.state.config.sources,
       schema: this.state.config.fieldSchema,
       uuid: messageUUID,
-    } as ProcessSample);
+    } as ProcessSample;
+
+    try {
+      labelsWorker.postMessage(workerArgs, transfer);
+    } catch (error) {
+      // rarely we'll get a DataCloneError
+      // if one of the buffers is detached and we didn't catch it
+      // try again without transferring the buffers (copying them)
```

Comment:
Is guaranteeing  no `DataCloneError` errors possible? How does a bitmap on the main thread get detached before being transfered?

Reply:
Oh, I see this is mentioned in `AbstractLooker.loadSample`